python/mlcroissant/mlcroissant/_src/tests/graphs/0.8/distribution_bad_type/output.txt

```diff
@@ -1,2 +1,4 @@
 Found the following 1 error(s) during the validation:
-  -  "a-csv-table" should have an attribute "@type": "https://schema.org/FileObject" or "@type": "https://schema.org/FileSet". Got https://schema.org/WRONG_TYPE_HERE instead.
\ No newline at end of file
+  -  "a-csv-table" should have an attribute "@type": "https://schema.org/FileObject" or "@type": "https://schema.org/FileSet". Got https://schema.org/WRONG_TYPE_HERE instead.
+Found the following 1 warning(s) during the validation:
+  -  `distribution` is empty for this dataset. If this is not expected, this might be because of bad `@id`s in the jsonld. Note that valid `@id`s cannot contain any whitespace characters to be successfully parsed by the rdflib.
```

Comment:
This is 0.8 but still mentions `@id`

Reply:
In this case, the distribution is empty because the file in the distribution has a bad type and is not registered under `distribution` (as below). The error message just states that empty distributions *can* be due to to wrong `@id`s. As one can add `@id`s for 0.8 datasets too (even if they are not mandatory as in 1.0), I added the error message for both versions. Do you think we should have it only for 1.0?